lib/src/pages/yaru_section.dart

```diff
@@ -45,6 +45,7 @@ class YaruSection extends StatelessWidget {
       padding: padding,
       margin: margin,
       child: Column(
+        mainAxisSize: MainAxisSize.min,
```

Comment:
Could you make it a parameter with this as default? Maybe there are cases where we want it to expand?

Reply:
you can expand the content as you wish. it's intentionally wrapped with flexible